docs/articles/documentation/test-api/selecting-page-elements/dom-node-snapshots.md

```diff
@@ -53,6 +53,8 @@ Property | Type | Description
 `offsetLeft` | Number | The number of pixels that the upper left corner of the element is offset by to the left within the `offsetParent` node. See [HTMLElement.offsetLeft](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/offsetLeft).
 `offsetTop` | Number | The number of pixels that the upper left corner of the element is offset by to the top within the `offsetParent` node. See [HTMLElement.offsetTop](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/offsetTop).
 `offsetWidth` | Number | The width of the element including vertical padding and borders. See [HTMLElement.offsetWidth](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/offsetWidth).
+`selected` | Boolean | Indicates that `<option>` element is currently selected. See [HTMLOptionElement](https://developer.mozilla.org/en-US/docs/Web/API/HTMLOptionElement).
```

Comment:
`selected` | Boolean | Indicates that `<option>` element is currently selected. For other elements, `undefined`.  See [HTMLOptionElement](https://developer.mozilla.org/en-US/docs/Web/API/HTMLOptionElement).
Is the '  See HTMLOptionElement.' sentence unnecessary in that case ?